spring-integration-core/src/main/java/org/springframework/integration/transformer/EncodingPayloadTransformer.java

```diff
@@ -36,7 +36,7 @@ public EncodingPayloadTransformer(Codec codec) {
 	}
 
 	@Override
-	protected byte[] transformPayload(T payload) throws Exception {
+	protected byte[] transformPayload(T payload) throws Exception {//NOSONAR
```

Comment:
Got it! Thank you for explanation!
That was my first experience with SonarQube, so any its comment is important for me :smile: 
Excuse me, if I introduced so much noise.